lib/dry/system/errors.rb

```diff
@@ -58,5 +58,14 @@ def initialize(name)
         super("component identifier #{name.inspect} must be a symbol")
       end
     end
+
+    # Error raised when trying to stop a component that hasn't started yet
+    #
+    # @api public
+    ComponentNotStartedError = Class.new(StandardError) do
+      def initialize(component_name)
+        super("You can not stop a component +#{component_name}+ has not been started.")
```

Comment:
@GustavoCaso Would you mind changing it to `"component +#{component_name}+ has not been started"`?

Reply:
Sure thing. Sorry if I did not removed in the last commit, I understood only removing the tip part

Reply:
@timriley done